pkg/controller/provider/validation.go

```diff
@@ -257,6 +257,10 @@ func (r *Reconciler) testConnection(provider *api.Provider, secret *core.Secret)
 		return nil
 	}
 	rl := container.Build(nil, provider, secret)
+	// Status code returns
+	// 0 - ignoring (unrealted connection fail)
+	// 200 - ok
+	// 401 - unauthorized
```

Comment:
I didn't mean to document http codes but rather to explain what are we doing and why do we want this special behavior for "Unauthorized"